src/Yasumi/Provider/Bosnia.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * This file is part of the Yasumi package.
+ *
+ * Copyright (c) 2015 - 2018 AzuyaLabs
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @author Sacha Telgenhof <stelgenhof@gmail.com>
+ */
+
+namespace Yasumi\Provider;
+
+use DateTime;
+use DateTimeZone;
+use Yasumi\Holiday;
+
+/**
+ * Provider for all holidays in Croatia.
+ *
+ * @author Karlo Mikus <contact@karlomikus.com>
+ */
+class Bosnia extends AbstractProvider
+{
+    use CommonHolidays, ChristianHolidays;
+
+    /**
+     * Code to identify this Holiday Provider. Typically this is the ISO3166 code corresponding to the respective
+     * country or sub-region.
+     */
+    const ID = 'BS';
```

Comment:
According to https://en.wikipedia.org/wiki/ISO_3166-2:BA, the ISO3166-2 code for Bosnia and Herzegovina is 'BA'.